src/main/java/frc/robot/commands/BalancerCommand.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Balancer;
+
+/** An example command that uses an example subsystem. */
+public class BalancerCommand extends CommandBase {
+
+  private final Balancer m_autobal;
+  public double returnPower;
+
+  /**
+   * Creates a new ExampleCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public BalancerCommand(Balancer subsystem) {
+    m_autobal = subsystem;
```

Comment:
so i think the intent of this command-based structure is for the subsystem to be the drivetrain, and all the logic for the balancer goes in the execute() method (which gets called similarly to the periodic() method of a subsystem). so essentially we do not need a balancer subsystem, which makes sense because there is no physical "balancer mech"